docs/src/main/sphinx/release/release-464.md

```diff
@@ -0,0 +1,41 @@
+# Release 464 (30 Oct 2024)
+
+## General
+
+* {{breaking}} Require JDK 23 to run Trino. ({issue}`21316`)
+* Add the [](/connector/faker) to allow easy generation of data. ({issue}`23691`)
+* Add the Vertica connector. ({issue}`23948`)
+* Rename the
+  `fault-tolerant-execution-eager-speculative-tasks-node_memory-overcommit`
+  configuration property to
+  `fault-tolerant-execution-eager-speculative-tasks-node-memory-overcommit`.
+  ({issue}`23876`)  
+
+## Accumulo connector
+
+* {{breaking}} Remove the Accumulo connector. ({issue}`23792`)  
+
+## Delta Lake connector
+
+* Fix S3 file listing of buckets that enforce requester pays. ({issue}`23906`)
+
+## Hive connector
+
+* Use the `hive.metastore.partition-batch-size.max` catalog configuration
+  property value in the `sync_partition_metadata` procedure. Change the default
+  batch size from 1000 to 100. ({issue}`23895`)
+* Fix S3 file listing of buckets that enforce requester pays. ({issue}`23906`)
+
+## Hudi connector
+
+* Fix S3 file listing of buckets that enforce requester pays. ({issue}`23906`)
+
+## Iceberg connector
+
+* Improve performance of `OPTIMIZE` on large partitioned tables. ({issue}`10785`)
+* Rename the `iceberg.expire_snapshots.min-retention` configuration property to
```

Comment:
Both of these renames are breaking, too, right?

Reply:
no .. there is a defined legacy config with the old name